### ISSUE TITLE
x64 build

### ISSUE THREAD

**Issue by javiergutierrezchamorro** [ivan386/Shareaza#9]
Will be 64 bit builds like in ex-Shareaza?

**Comment by javiergutierrezchamorro** [comment]
No updates?

**Comment by siakc** [comment]
I don't understand the original question. Sorry.

**Comment by javiergutierrezchamorro** [comment]
@siakc question is if there will be also x64 builds available.

**Comment by javiergutierrezchamorro** [comment]
> @siakc question is if there will be also x64 builds available.

Is it clear now?

**Comment by siakc** [comment]
> 

Yes it is. This project is abandoned. You may need to compile it yourself as per instructions in the readme file. Sorry for the inconvenience.

**Comment by javiergutierrezchamorro** [comment]
@ivan386 any thoughts?

**Comment by ivan386** [comment]
I  agree with the siakc. 64 build broken in my code. And i don't want to work on that project.

**Comment by javiergutierrezchamorro** [comment]
Thank you for replying @ivan386

**Comment by ansani** [comment]
For reference, I forked the project and made x64 builds here: https://github.com/ansani/Shareaza/releases

**Comment by javiergutierrezchamorro** [comment]
> For reference, I forked the project and made x64 builds here: https://github.com/ansani/Shareaza/releases

Really appreciate it @ansani

**Comment by ghost** [comment]
> For reference, I forked the project and made x64 builds here: https://github.com/ansani/Shareaza/releases

What exactly is recommended? As far as I understand there are so many forks. Which one to use? Is your fork the continuation of the Shareaza project or is this just a copy of the orginal with changes? Will this fork become official at some point? So many questions :)

Why was this project abandoned in the first place? So much work was done to make Shareaza a successful project. 
Then I always regret that the project stops somewhere. The latest official update was in 2017.

By the way, I use the x64 version from 2017 and I haven't had any problems with it.

**Comment by javiergutierrezchamorro** [comment]
@prenniee you can take a look at the commit log for what has changed:
https://github.com/ansani/Shareaza/commits/master

**Comment by ghost** [comment]
@javiergutierrezchamorro 
You mean what was changed compared to the official release or compared to the master branch of Ansani? Thanks!

**Comment by ansani** [comment]
> > For reference, I forked the project and made x64 builds here: https://github.com/ansani/Shareaza/releases
> 
> What exactly is recommended? As far as I understand there are so many forks. Which one to use? Is your fork the continuation of the Shareaza project or is this just a copy of the orginal with changes? Will this fork become official at some point? So many questions :)

My project is just a fork with some changes and improvements over the official client. My fork will never become the new official release :)

> 
> Why was this project abandoned in the first place? So much work was done to make Shareaza a successful project. Then I always regret that the project stops somewhere. The latest official update was in 2017.

You can find the full story of Shareaza client/domain/sourcecode here: https://en.wikipedia.org/wiki/Shareaza
